frontend/app/src/scenes/Home/styles.css

```diff
@@ -56,3 +60,19 @@
 .mainTitle {
     margin-top: 5px;
 }
+
+.closeButton {
+    position: absolute;
+    right: 0.8rem;
+    color: red;
```

Comment:
shall we use `danger` button instead of adding color here?  (in the current implementation, the button turn green at hover)